### DISCUSSION TITLE
Favicon not updating on the deployed site

### DISCUSSION THREAD

**Discussion by dkubatko** [vercel/community#6655] [Help]
### Summary

I have replaced the default favicon with my custom one, and locally it displays just fine if I delete `.next` and re-run the server. However, multiple commits after, the favicon on my deployed site is not updated. Yes, I cleared browser cache and it is not updated on multiple devices. What can I do here?

### Steps to Reproduce

1. Replace default favicon.ico
2. Push to Vercel
3. favicon not updated

**Comment by mrezabayat** [comment]
Same here
[hermes-group-w2ds-1tk0qq2xf-mohammads-projects-356f4c16.vercel.app](https://vercel.com/mohammads-projects-356f4c16/hermes-group-w2ds/4MX4bAmnNAbodevvwHV3hA73BTJd)

**Comment by istrk** [comment]
Had the same issue. vercel --force and then vercel --prod and then clearing browsing data in the browser helped me

**Comment by Th1nhNg0** [comment]
confirm I had the same issue, I has to delete it, deploy, then add again

**Comment by kirankulkarni** [comment]
@Th1nhNg0 suggestion worked for me. It's not convenient

**Comment by amyegan** [comment]
Deploying without the build cache worked for my example app. You can can bypass the build cache by [redeploying](https://vercel.com/docs/deployments/managing-deployments#redeploy-a-project) or by using [`vercel --force --prod`](https://vercel.com/docs/cli/deploy#force) from your local repo.

[vercel.com/docs/deployments/troubleshoot-a-build#managing-build-cache](https://vercel.com/docs/deployments/troubleshoot-a-build#managing-build-cache)

**Reply by imyanice** [reply]
> Deploying without the build cache worked for my example app. You can can bypass the build cache by [redeploying](https://vercel.com/docs/deployments/managing-deployments#redeploy-a-project) or by using [`vercel --force --prod`](https://vercel.com/docs/cli/deploy#force) from your local repo.
> 
> [vercel.com/docs/deployments/troubleshoot-a-build#managing-build-cache](https://vercel.com/docs/deployments/troubleshoot-a-build#managing-build-cache)

hi, this did not fix it for me after a week.
https://murderers.lol
